sqlstore/src/main/resources/db/migration/V8012__sequencer_update.sql

```diff
@@ -0,0 +1,25 @@
+DELETE FROM SequencingParameters WHERE SequencingParameters.name = 'v4 1×136';
+
+DELETE FROM Platform WHERE Platform.name IN ('Solid','LS454','IonTorrent','OxfordNanopore');
```

Comment:
Agreed. I thought the plan was to filter them based on installed sequencers in the UI, not the DB.